tests/python/frontend/to_onnx/test_to_onnx.py

```diff
@@ -0,0 +1,262 @@
+    input_names ={}
+    for input, data in zip(sess.get_inputs(), input_data):
+        input_names[input.name] = data
+    output_name = sess.get_outputs()[0].name
+    res = sess.run([output_name], input_names)
+    return res[0]
+
+
+def do_relay_inference(func, data_tuple):
+    target = 'llvm'
+    ctx = tvm.context('llvm', 0)
+    intrp = relay.create_executor("graph", ctx=ctx, target=target)
+    relay_res = intrp.evaluate(func)(*data_tuple)
+    return relay_res.asnumpy()
+
+
+def test_add():
+    dtype = 'float32'
+    t1 = relay.TensorType((5, 10, 5))
+    t2 = relay.TensorType((5, 10, 5))
+    x = relay.var("x", t1, dtype=dtype)
+    y = relay.var("y", t2, dtype=dtype)
+    z = relay.add(x, y)
+    func = relay.Function([x, y], z)
+
+    x_data = np.random.rand(5, 10, 5).astype(dtype)
+    y_data = np.random.rand(5, 10, 5).astype(dtype)
+
+    relay_res = do_relay_inference(func, (x_data, y_data))
+    onnx_res = do_onnx_inference(func_to_onnx(func, 'add'), [x_data, y_data])
+
+    np.testing.assert_allclose(relay_res, onnx_res)
+
+
+def test_bias_add():
+    for dtype in ['float16', 'float32']:
+        xshape = (10, 2, 3, 4)
+        bshape = (2,)
+        rtol = 1e-2 if dtype is 'float16' else 1e-5
+        x = relay.var("x", shape=xshape, dtype=dtype)
+        bias = relay.var("bias", dtype=dtype)
+        z = relay.nn.bias_add(x, bias)
+        func = relay.Function([x, bias], z)
+
+        x_data = np.random.uniform(size=xshape).astype(dtype)
+        y_data = np.random.uniform(size=bshape).astype(dtype)
+
+        relay_res = do_relay_inference(func, (x_data, y_data))
+        onnx_res = do_onnx_inference(func_to_onnx(func, 'test_bias_add'), [x_data, y_data])
+
+        np.testing.assert_allclose(relay_res, onnx_res, rtol=rtol)
+
+
+def test_conv2d():
+    def run_test_conv2d(dtype, out_dtype, scale, dshape, kshape,
+                        padding=(1, 1),
+                        groups=1,
+                        dilation=(1, 1),
+                        **attrs):
+
+        x = relay.var("x", shape=dshape, dtype=dtype)
+        w = relay.var("w", dtype=dtype)
+        y = relay.nn.conv2d(x, w,
+                            padding=padding,
+                            dilation=dilation,
+                            groups=groups,
+                            **attrs)
+        func = relay.Function([x, w], y)
+        data = np.random.uniform(-scale, scale, size=dshape).astype(dtype)
+        kernel = np.random.uniform(-scale, scale, size=kshape).astype(dtype)
+        relay_res = do_relay_inference(func, (data, kernel))
+        onnx_res = do_onnx_inference(func_to_onnx(func, 'test_conv2d'), [data, kernel])
+
+        tvm.testing.assert_allclose(relay_res, onnx_res, rtol=1e-5, atol=1e-5)
+
+
+    dshape = (1, 32, 18, 18)
+    kshape = (32, 1, 3, 3)
+    run_test_conv2d("float32", "float32", 1, dshape, kshape,
+                    padding=(1, 1), channels=32, groups=32, kernel_size=(3, 3))
```

Comment:
add some test cases for padding besides (1,1) and kernal_size besize (3, 3)

Reply:
added